src/components/base/link/Link.js

```diff
@@ -59,15 +86,14 @@ function Link({
         ref={linkRef}
         data-cy={dataCy}
         target={target}
-        onClick={(e) => {
-          if (onClick) {
-            if (!href) {
-              e.preventDefault();
-            }
-            onClick(e);
-          }
+        onClick={(e) => onClickWrapper(onClick, href, e)}
+        onKeyDown={(e) => {
+          onKeyDown
+            ? onKeyDown(e)
+            : e?.key === "Enter" || e?.key === " "
```

Comment:
Ville nok ikke sætte spacebar på som default

https://www.clairecodes.com/blog/2019-11-06-the-roles-of-spacebar-and-enter-in-keyboard-navigation/